src/modules/workflows/enums.js

```diff
@@ -22,6 +22,33 @@ const WorkflowKeysEnum = {
   assignee: 'assignee'
 };
 
+const WorkflowAuditLogKeysEnum = {
+  id: 'id',
+  assignee: 'assignee',
+  bulletPoints: 'bulletPoints',
+  competitiveCyclistDescription: 'competitiveCyclistDescription',
+  competitiveCyclistTopline: 'competitiveCyclistTopline',
+  detailedDescription: 'detailedDescription',
+  genus: 'genus',
+  // harmonizingData: 'harmonizingData',
```

Comment:
dont commit commented lines.
Remove if not needed

Reply:
done